scripts/crda.sh

```diff
@@ -13,8 +13,6 @@ output_file_path="$2"
 pkg_installation_directory_path="$3"
 consumer="$4"
 
-# Setting the package installation directory path
-export PYTHONPATH=$pkg_installation_directory_path
```

Comment:
Remove this line as this line can be added in Tekton Task and the script can be the same script can be used for all ecosystems.

Reply:
To avoid any failure for consumer of images, this line should not be removed unless it is added in all consumers implementation like Task script and GH Action script.